app_dart/lib/src/service/datastore.dart

```diff
@@ -114,10 +114,15 @@ class DatastoreService {
   /// If startFrom is nil, starts from the latest available record.
   /// [startFrom] to be implemented...
   Stream<TimeSeriesValue> queryRecentTimeSeriesValues(TimeSeries timeSeries,
-      {int limit = 1500, String startFrom, String branch = 'master'}) {
+      {int limit = 1500,
```

Comment:
Please add a constant for this value it will make easier to change in a single place.

Reply:
Added.